packages/gestalt/src/Button.jsdom.test.js

```diff
@@ -8,28 +8,17 @@ describe('Button', () => {
     const mockOnClick = jest.fn<
       [
         {|
-          dangerouslyDisableOnNavigation: () => void,
-          event:
-            | SyntheticMouseEvent<HTMLButtonElement>
-            | SyntheticMouseEvent<HTMLAnchorElement>
-            | SyntheticKeyboardEvent<HTMLAnchorElement>
-            | SyntheticKeyboardEvent<HTMLButtonElement>,
+          event: SyntheticMouseEvent<HTMLButtonElement> | SyntheticKeyboardEvent<HTMLButtonElement>,
         |},
       ],
       void,
     >();
-    render(<Button text="ButtonText" onClick={mockOnClick} />);
-    screen.getByText('ButtonText').click();
+    const { getByText } = render(<Button text="ButtonText" onClick={mockOnClick} />);
+    // eslint-disable-next-line testing-library/prefer-screen-queries -- Please fix the next time this file is touched!
+    getByText('ButtonText').click();
```

Comment:
this is a regression. Internal: remove all eslint-disable-next-line testing-library/prefer-screen-queries disables from unit tests (https://github.com/pinterest/gestalt/pull/3182) - [Preview link](https://deploy-preview-3182--gestalt.netlify.app/?devexample=true)

Reply:
Oops, I think a rogue copypaste deleted this. Fixed.